lib/hawkular/base_client.rb

```diff
@@ -146,8 +146,14 @@ def token_header
     end
 
     def tenant_header
-      @options[:tenant].nil? ? { 'Hawkular-Tenant' => 'hawkular' } : { :'Hawkular-Tenant' => @options[:tenant],
-                                                                       'tenantId'         => @options[:tenant] }
+      if @options[:tenant].nil?
+        { 'Hawkular-Tenant' => 'hawkular' }
```

Comment:
shouldn't be the default. user should pass the needed tenant (similar to that you won't set user/password default values)